main/streams/plain_wrapper.c

```diff
@@ -125,11 +125,12 @@ typedef struct {
 	FILE *file;
 	int fd;					/* underlying file descriptor */
 	unsigned is_process_pipe:1;	/* use pclose instead of fclose */
-	unsigned is_pipe:1;			/* don't try and seek */
```

Comment:
It would make sense not to reorder the fields.

Reply:
Yes, makes sense.